x-pack/plugins/security/server/lib/authorization/privileges.ts

```diff
@@ -57,6 +57,7 @@ export function privilegesFactory(
             actions.app.all,
             ...actions.savedObject.readOperations(validSavedObjectTypes),
             ...featuresPrivilegesBuilder.getUIReadActions(features),
+            ...featuresPrivilegesBuilder.getManagementReadActions(features),
```

Comment:
Is there a reason why this wasn't also added to the spaces read privilege?

Reply:
hmmmm.... no? I don't remember arguing with myself about this, so I probably just overlooked it. Thanks for catching, I'll add.

Reply:
fixed in https://github.com/elastic/kibana/pull/27055/commits/9faa35669183a0b66949ba2fbd4cf8aad47ce06c